frontend/coreproject/src/routes/+layout.svelte

```diff
@@ -192,7 +192,7 @@
 					{/each}
 				</div>
 
-				<div class="mt-20 flex flex-col items-center md:gap-5 2xl:gap-9">
+				<div class="md:mt-10 2xl:mt-20 flex flex-col-reverse items-center md:gap-5 2xl:gap-9">
```

Comment:
> flex-col-reverse 

Whats this?

Reply:
I tried placing settings on bottom, it looks nice there
Dont like this ?

Reply:
![image](https://user-images.githubusercontent.com/61817579/230699588-43f93a4e-1c9b-40de-aad6-fb4d4d0e0adf.png)


See the problem?